app/Http/Middleware/AddContentLength.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Gladiator\Http\Middleware;
+
+use Closure;
+
+class AddContentLength
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        $response = $next($request);
+        $response->header('Content-Length', strlen(serialize($response)));
```

Comment:
I think we want `$response->getContent()`, since the `serialize()` function would give us the length of the PHP serialized object, not the HTTP response itself. And let's use either `mb_strlen` or Laravel's `Str::length` so that we don't run into issues with UTF-8 characters in the response (like emoji! ✨)

```php
$response->header('Content-Length', Str::length($response->getContent()));
```